CHANGELOG.md

```diff
@@ -0,0 +1,1242 @@
+# Changelog
```

Comment:
Will this CHANGELOG be automatically synced with https://github.com/meta-llama/llama-stack/releases?

Reply:
Nope, it's manual right now

Reply:
got it, shall we file an issue to automate this? learning from the argo project you referred to: https://github.com/argoproj/argo-workflows/blob/fc5bcc8665b1b2b73f0be09e63e978b6c0c3c746/.github/workflows/changelog.yaml#L2

Reply:
I might have a script for this. Stay tuned